library/iFixit/Matryoshka/APCu.php

```diff
@@ -78,6 +82,12 @@ public function deleteMultiple(array $keys) {
       // when you provide an array of keys, it provides an array of errors (if any)
       // so the only successful case is no errors (empty array).
       $ret = apcu_delete($keys);
+      /**
+       * In the docs, the apcu_delete returns an array if passed one.
+       * Ref: https://www.php.net/manual/en/function.apcu-delete.php
+       * Psalm gets this wrong.
+       * @psalm-suppress InvalidArgument
```

Comment:
Is type-stubbing a better option here?

Reply:
Its nasty. #wontfix. The "real" fix would touch every other backend.

Reply:
I think type-stubbing would be more correct, but I'm not clear on what the return value of `apcu_delete` indicates when given more than one key. The [docs](https://www.php.net/manual/en/function.apcu-delete.php) seem to say that it returns the equivalent of `array_keys` of its argument, which seems boring. If that's correct, then I don't think we want to be returning its result at all, because it doesn't actually tell us anything. Given that this appears to be a pointless return value anyway, I'd also be fine with just dropping the return type on this globally, with no need to stub.